docs/spec/tempo.grafana.com_tempostacks.yaml

```diff
@@ -182,6 +183,9 @@ spec:                                    # TempoStackSpec defines the desired st
         monitorTab:                      # MonitorTab defines the monitor tab configuration.
           enabled: false                 # Enabled enables the monitor tab in the Jaeger console. The PrometheusEndpoint must be configured to enable this feature.
           prometheusEndpoint: ""         # PrometheusEndpoint defines the endpoint to the Prometheus instance that contains the span rate, error, and duration (RED) metrics. For instance on OpenShift this is set to https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+        security:                        # Oauth defines the options for the oauth proxy used to protect jaeger UI
```

Comment:
I wonder if we should use a more descriptive name than `security`. Maybe `authentication`?

Reply:
Changed to authenticate!